config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
-  devise_for :users
+
+  devise_for :users, :controllers => { :invitations => 'users_controller/invitations' }
```

Comment:
Be nice to use modern Ruby syntax here i.e. `devise_for :users, controllers: { invitations: 'users_controller/invitations' }`